contracts/interface/ILimitOrderBook.sol

```diff
@@ -6,7 +6,8 @@ interface ILimitOrderBook {
     // Do NOT change the order of enum values because it will break backwards compatibility
     enum OrderType {
         LimitOrder,
-        StopLimitOrder
+        StopLimitOrder,
```

Comment:
Wonder why aren't they `stopLoss` & `takeProfit`? I thought these two are symmetric

Reply:
Haha, I simply follow the naming from FTX.

![Screen Shot 2022-06-06 at 14 33 05](https://user-images.githubusercontent.com/652070/172108019-93a84434-aa7a-4bbb-9079-8ba660ae16f3.png)

Sure, I will rename `StopLimitOrder` to `StopLossLimitOrder` in contract.